admin/tool/policy/contactadmin.php

```diff
@@ -0,0 +1,79 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Display administrator's contact details to a digital minor.
+ *
+ * @package     tool_policy
+ * @category    admin
+ * @copyright   2018 Mihail Geshoski <mihail@moodle.com>
+ * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+require(__DIR__.'/../../../config.php');
+require_once($CFG->libdir . '/authlib.php');
+
+if (!$authplugin = signup_is_enabled()) {
+    print_error('notlocalisederrormessage', 'error', '', 'Sorry, you may not use this page.');
+}
+
+//HTTPS is required in this page when $CFG->loginhttps enabled
+$PAGE->https_required();
```

Comment:
This is deprecated in Moodle 3.4. I don't think it makes sense to introduce it even into 3.3 version. But maybe there is a good reason for that?

Reply:
Thanks David, I wasn't aware of that.